doc/nrf/ug_matter_configuring.rst

```diff
@@ -6,19 +6,15 @@ Configuring Matter
 When you build any of the available Matter samples to the supported development kits, you automatically build the Matter stack for the nRF Connect platform.
 The development kit and the application running Matter stack that is programmed on the development kit together form the Matter accessory device.
 
-.. raw:: html
 
-   <div style="text-align: center;">
-   <iframe width="560" height="315" src="https://www.youtube.com/embed/XWWJYraNZPs" title="Matter development environment video overview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-   </div>
 
-Currently, the |NCS| only supports Matter stack that is built on top of a low-power, 802.15.4-compatible Thread network.
-To commission and control the Matter accessory device remotely over such a network, you need to set up a Matter controller on a mobile or a PC.
+Currently, the |NCS| only supports Matter stack that is built on top of a low-power, 802.15.4-compatible Thread network and over the Wi-Fi network.
```

Comment:
```suggestion
The |NCS| supports Matter stack that is built on top of a low-power, 802.15.4-compatible Thread network, or over the Wi-Fi network.
```
We now support 2 out of 3 networking technologies covered by Matter, so we no longer need to be shy and say "only" :)